src/app/shared/search/search-results/search-results-skeleton/search-results-skeleton.component.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AsyncPipe,
+  NgForOf,
+} from '@angular/common';
+import {
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
+import { Observable } from 'rxjs';
+
+import { SearchService } from '../../../../core/shared/search/search.service';
+import { ViewMode } from '../../../../core/shared/view-mode.model';
+import { hasValue } from '../../../empty.util';
+
+@Component({
```

Comment:
Please add some TypeDocs or a comment to describe the purpose of this component.  We require that for all new components.